commcare_connect/templates/opportunity/user_visit_verification_table.html

```diff
@@ -0,0 +1,55 @@
+{% load django_tables2 %}
+
+<form class="flex flex-col w-full gap-2"
```

Comment:
This should still work if we replace `form` with `div`, right?

Reply:
The form automatically includes any inputs that are inside it when htmx does a request. We will need to manually add any inputs in the `hx-params` attribute if we change this to a div.